pymc_marketing/clv/models/beta_geo.py

```diff
@@ -437,6 +437,71 @@ def expected_probability_alive(
             "chain", "draw", "customer_id", missing_dims="ignore"
         )
 
+    def expected_probability_no_purchase(
+        self,
+        t: int,
+        data: pd.DataFrame | None = None,
+    ) -> xarray.DataArray:
+        r"""Compute the probability a customer with history frequency, recency, and T
+        will have 0 purchases in the period (T, T+t].
+
+        The data parameter is only required for out-of-sample customers.
+
+        Adapted from Section 5.3 in Bruce Hardie's notes [1]_ for the special case when y = 0
```

Comment:
Can you please link the number of the equation in the reference please?

Reply:
Can we also add this reference to the [model class documentation](https://github.com/pymc-labs/pymc-marketing/blob/main/pymc_marketing/clv/models/beta_geo.py#L138)? Don't forget the numerical citation in the "First introduced by Fader. . . " paragraph.